packages/terra-modal/src/ModalContent.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { PropTypes } from 'react';
+import FocusTrap from 'focus-trap-react';
+import ModalOverlay from './ModalOverlay';
+
+const propTypes = {
+  ariaLabel: PropTypes.string.isRequired,
+  children: PropTypes.node.isRequired,
+  classNameModal: PropTypes.string,
+  classNameOverlay: PropTypes.string,
+  closeOnOutsideClick: PropTypes.bool,
+  closePortal: PropTypes.func,
+  role: PropTypes.string,
+};
+
+const defaultProps = {
+  ariaLabel: null,
+  children: null,
+  classNameModal: null,
+  classNameOverlay: null,
+  closeOnOutsideClick: true,
+  role: 'dialog',
+};
+
+
+class ModalContent extends React.Component {
+
+  componentDidMount() {
+    // Disable background scrolling
+    document.body.style.overflow = 'hidden';
+  }
+
+  componentWillUnmount() {
+    // Enable background scrolling
+    document.body.style.overflow = null;
```

Comment:
I think could cause unwanted side effects. The base layout every Orion application uses currently has body overflow set to hidden. This will result in removing that style and potentially causing a lot of issues with the layout. 

https://github.cerner.com/orion/orion-rails-layouts/blob/778b1c4973bfbeab416e228f66f71ab75c737d8e/app/views/orion/rails/layouts/_base.html.haml#L25

https://github.cerner.com/orion/orion-rails/blob/c26aa2eb622708549f70bcd84a9f95adf3713d7a/app/assets/stylesheets/orion/layouts/base.less#L15